sdk/src/client/api/block_builder/pow.rs

```diff
@@ -8,11 +8,11 @@ use crate::pow::miner::{Miner, MinerBuilder, MinerCancel};
 #[cfg(target_family = "wasm")]
 use crate::pow::wasm_miner::{SingleThreadedMiner, SingleThreadedMinerBuilder};
 use crate::{
-    client::{Client, Error, Result},
+    client::{ClientInner, Error, Result},
     types::block::{parent::Parents, payload::Payload, Block, BlockBuilder, Error as BlockError},
 };
 
-impl Client {
+impl ClientInner {
```

Comment:
Just checking but all the methods on Client that you are now putting under ClientInner will still be available to Client because of the Deref, right?

Reply:
Yep